### DISCUSSION TITLE
hackintosh/updates/2021/09/06/acidanthera-september

### DISCUSSION THREAD

**Discussion by giscus** [dortania/dortania.github.io#22] [Comments]
# hackintosh/updates/2021/09/06/acidanthera-september

A message from vit9696 Changelogs

https://dortania.github.io/hackintosh/updates/2021/09/06/acidanthera-september.html

**Comment by flagersgit** [comment]
*goodest*

**Reply by perez987** [reply]
Thank you!!!

**Comment by ThatStella7922** [comment]
epic

**Comment by AquaHacker** [comment]
THANK！！！

**Comment by narcyzzo** [comment]
Dope

**Comment by akashm10** [comment]
thank you so much!

**Comment by hh724330184** [comment]
Thanks a lot!!!!

**Comment by jcckg** [comment]
# nice

**Comment by kaoskinkae64** [comment]
perfect

**Comment by andy222-ux** [comment]
nice
nice

**Comment by pnxl** [comment]
very epic

**Reply by ghost** [reply]
true

**Comment by SKiD12** [comment]
a noob question - is there any way (and reason) to update existing OpenCore installation. Thanks :)

**Reply by thieuayr** [reply]
https://dortania.github.io/OpenCore-Post-Install/universal/update.html#updating-opencore

**Reply by SKiD12** [reply]
thanks mate, I totally missed this topic 🙈
